forge/db/views/User.js

```diff
@@ -38,6 +38,9 @@ module.exports = {
         if (user.password_expired) {
             result.password_expired = true
         }
+        if (user.tcs_accepted) {
```

Comment:
We should only include this in the response if `app.settings.get('user:tcs-required')` is set.

Reply:
resolved in [9e58e97](https://github.com/flowforge/flowforge/pull/951/commits/9e58e97fa516f3faf262245ef094b0d4017a680f)